apb.yml

```diff
@@ -7,7 +7,7 @@ tags:
   - mobile-service
   - mobile-client-enabled
 metadata:
-  displayName: Mobile CI|CD
+  displayName: Mobile CI|CD - with proxy
```

Comment:
Wonder should we add some more info here into proxy details? As in what's its purpose etc